content/vgg-classifier-tutorial.md

```diff
@@ -0,0 +1,320 @@
+Title: Keras, Make your own image classifier
+Date: 2016-03-30
+Category: Tutorial
+Author: Gregory Senay
+
+
+This tutorial explains how to create image classifier, from an ImageNet pre-trained model, your own model on your own dataset.
```

Comment:
"how to create *an* image classifier, using a model pre-trained on ImageNet and fine-tuned over your own dataset"